src/utils/url.ts

```diff
@@ -43,12 +43,16 @@ export const openRoomUrl = (url: string) => {
 export const openUrl = (url: string) => {
   if (!isValidUrl(url)) {
     Bugsnag.notify(
-      new Error(`Invalid URL ${url} on page ${window.location.href}; ignoring`),
+      // new Error(`Invalid URL ${url} on page ${window.location.href}; ignoring`),
+      new Error(
+        `Invalid URL ${url} on page ${window.location.href}; allowing for now (workaround)`
+      ),
       (event) => {
         event.addMetadata("utils/url::openUrl", { url });
       }
     );
-    return;
+    // @debt keep the checking in place so we can debug further, but don't block attempts to open
+    // return;
```

Comment:
I'm explicitly choosing to keep this commented code here as this should be a temporary workaround, after which it should be able to be restored (+whatever the bug fix is)